src/include/target/DIY_900_RX_STM32.h

```diff
@@ -0,0 +1,28 @@
+#ifndef DEVICE_NAME
+#define DEVICE_NAME "ELRS 900RX"
+#endif
+
+#define TARGET_DIY_900_RX_STM32
+
+// GPIO pin definitions
+#define GPIO_PIN_NSS         PA4
+#define GPIO_PIN_MOSI        PA7
+#define GPIO_PIN_MISO        PA6
+#define GPIO_PIN_SCK         PA5
+
+#define GPIO_PIN_DIO0        PA0
+#define GPIO_PIN_DIO1        PA1
+#define GPIO_PIN_RST         PA2
+#define GPIO_PIN_BUSY        PA3
+
+#define GPIO_PIN_RCSIGNAL_RX PA10 // USART1
+#define GPIO_PIN_RCSIGNAL_TX PA9  // USART1
+
+#define GPIO_PIN_DEBUG_RX    PB11 // USART3
+#define GPIO_PIN_DEBUG_TX    PB10 // USART3
+
+#define GPIO_PIN_LED_GREEN   PC13
+#define GPIO_LED_GREEN_INVERTED 1
+
+// Output Power - default for SX127x
+#define POWER_OUTPUT_FIXED 15 //MAX power for 900 RXes
```

Comment:
This can be updated to have options for 10, 25, and 50 mW. https://github.com/ExpressLRS/ExpressLRS/pull/1981

Reply:
How do I change it? I searched for `power_output_fixed` in the repo and found the same line in several places.

Reply:
It looks like `POWERMGNT.cpp` handles the power levels, and it only considers the 2.4 SX1280 chip

Reply:
change to the following
```cpp
#define MinPower                PWR_10mW
#define MaxPower                PWR_50mW
#define DefaultPower            PWR_50mW
#define POWER_OUTPUT_VALUES     {8,12,15}
```

Reply:
Okay, updated. What does 8,12, and 15 correspond to?

Reply:
Isn't it possible for the RFM95W to run at 100mW? Can I increase the MAX value, and what power value does that correspond with?

Reply:
Only at 1% duty cycle, which ELRS is WAY over and up around 80%..